src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -66,6 +71,22 @@ private Map<String, Map<String, BindPiecesCollection>> groupBindPieceByPoLineId(
     return Map.of(poLineId, bindPieceMap);
   }
 
+  public Future<Void> removeBinding(String pieceId, RequestContext requestContext) {
+    logger.debug("removeBinding:: Removing binding for piece: {}", pieceId);
+    return pieceStorageService.getPieceById(pieceId, requestContext)
+      .compose(piece -> {
+        // Populate piecesByLineId to accommodate helper methods
+        piecesByLineId = Map.of(piece.getPoLineId(), Collections.singletonMap(piece.getId(), null));
+        // Remove binding for the piece
+        piece.withBindItemId(null).withIsBound(false);
+        // Update piece record
+        return removeForbiddenEntities(requestContext)
+          .compose(v -> getValidPieces(requestContext))
+          .compose(piecesGroupedByPoLine -> storeUpdatedPieceRecords(piecesGroupedByPoLine, requestContext))
+          .mapEmpty();
```

Comment:
what will happen to title -> bindItemIds section? I assume itemId will be removed from this field

Reply:
Well, the item still exists but the piece is not associated it and is not bound anymore. @SerhiiNosko What should we have as an expected behaviour in this context?

Reply:
Should be resolved with latest changes